spec/rubocop/cli/auto_gen_config_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 require 'timeout'
 
-RSpec.describe RuboCop::CLI, :isolated_environment do
-  subject(:cli) { described_class.new }
+RSpec.describe 'RuboCop::CLI --auto-gen-config', :isolated_environment do # rubocop:disable RSpec/DescribeClass
```

Comment:
Unrelated to this PR. It seems that our `RSpec/FilePath` has two deficiencies:
 - it ignores non-constant docstring. If you happen to change it to `describe 'foo bar'`, it ignores it. Not sure why we've decided to do it this way
 - implies that the second argument is a method name in `describe RuboCop::CLI, '--auto-gen-config'`, strips dashes and suggests `autogenconfig` to be a part of the file name. In this case, this doesn't apply so well. Writing it like this triggers `RSpec/DescribeMethod` though

I can file a ticket if you think it's important